packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -19,15 +19,31 @@ const initialState: CoinjoinState = {
     clients: {},
 };
 
-const createAccount = (draft: CoinjoinState, account: Account) => {
+const createAccount = (
+    draft: CoinjoinState,
+    { account, targetAnonymity }: Extract<Action, { type: typeof COINJOIN.ACCOUNT_CREATE }>,
```

Comment:
Please use something like `ReturnType<typeof createAccountAction>` that should be compatible with RTK. We are not going to use this union `Action` type with combination with redux-toolkit so it will be problem to migrate.

Reply:
normally i would do it if it would be the first occurrence, but since it's already in this format in develop im leaving this to further migration together with other reducers using `Action`